ui/util/constants.ts

```diff
@@ -44,5 +48,10 @@ export const isSignInEmailInput = (node: UiNode): node is NodeWithLabel =>
 export const isSignInWithPassword = (node: UiNode): node is NodeWithLabel =>
   node.meta.label?.id === ORY_LABEL_SIGN_IN_WITH_PASSWORD;
 
-export const isContinueWithPasswordReset = (node: UiNode): node is NodeWithLabel =>
+export const isContinueWithPasswordReset = (
+  node: UiNode,
+): node is NodeWithLabel =>
   node.meta.label?.id === ORY_LABEL_CONTINUE_PASSWORD_RESET;
+
+export const isSignInWithHardwareKey = (node: UiNode): node is NodeWithLabel =>
+  node.meta.label?.id === ORY_LABEL_SIGN_IN_WITH_HARDWARE_KEY;
```

Comment:
nitpick(feel free to disregard): these two and the `isSignInWithPassword` look like they would belong to a different file

Reply:
I think they all belong here. Only the webauthn constants wouldn't fit, I moved them into another place just now.